app/forms/feedback.py

```diff
@@ -85,13 +75,11 @@ def __init__(self, *args, **kwargs):
             ("disagree", gettext("Disagree")),
             ("strongly disagree", gettext("Strongly disagree")),
         ]
+        self.demonstrate_why_org_funding.validators = [InputRequired(message=gettext("Select a score"))]
 
 
 class EndOfApplicationPage3Form(PrepopulatedForm):
-    understand_eligibility_criteria = RadioField(
-        label="How easy was it to understand the eligibility criteria for this fund?",
-        validators=[DataRequired(message="Select a score")],
-    )
+    understand_eligibility_criteria = RadioField()
```

Comment:
How these are getting the translations ?

Reply:
In here: https://github.com/communitiesuk/funding-service-design-frontend/pull/638/files#diff-5d730f4aa3603b9aa3d0d4102f16a8a8aced2a30207d259f853741e52bb9b18eR99